docs/iris/src/whatsnew/3.0.rst

```diff
@@ -17,20 +17,20 @@ This document explains the changes made to Iris for this release
 
    * We've finally dropped support for ``Python 2``, so welcome to ``Iris 3``
      and ``Python 3``!
-   * Our :ref:`documentation <whatsnew 3.0 docs>` has been refreshed,
-     restructured, revitalised and rehosted on `readthedocs`_,
-   * It's now easier than ever to :ref:`install Iris <installing_iris>`
-     as a user or a developer, and the newly revamped developers guide walks
-     you though how you can :ref:`get involved <development_where_to_start>`
-     and contribute to Iris,
    * We've extended our coverage of the `CF Conventions and Metadata`_ by
      introducting support for `CF Ancillary Data`_ and `Quality Flags`_,
    * Lazy regridding is now available for several regridding schemes,
-   * We've introduced a common metadata API to simplify and unify the
-     management of metadata across Iris,
+   * Managing and manipulating metadata within Iris is now easier and more
+     consistent thanks to the introduction of a new common metadata API,
    * Cube arithmetic has been significantly improved with regards to extended
```

Comment:
Could make "Cube arithmetic" link to the "Cube maths" section in the User Guide.  We know users are looking forward to getting the rather fabulously improved behaviour, and making it a link will make the words stand out in blue.